Others/Prime.java

```diff
@@ -0,0 +1,75 @@
+package Others;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class Prime {
+    private static class Pair<T, S>{
```

Comment:
Please use the built-in Pair class from the standard library.  See here http://www.baeldung.com/java-pairs